src/ui/server/controllers/insurance/account/password-reset/link-expired/index.ts

```diff
@@ -21,3 +29,40 @@ export const get = (req: Request, res: Response) =>
       BACK_LINK: req.headers.referer,
     }),
   });
+
+/**
+ * post
+ * Get an account by ID and call the API to send a password reset link email
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} "Password reset - link sent" page
+ */
+export const post = async (req: Request, res: Response) => {
+  try {
+    console.info('Posting account password reset - link expired form');
+
+    const urlOrigin = req.headers.origin;
+
+    const accountId = req.query.id;
+
+    if (!accountId) {
+      return res.redirect(PASSWORD_RESET_ROOT);
+    }
+
+    const account = await api.keystone.account.get(accountId);
```

Comment:
do we need a check for if account does not exist? or comes back as undefined?

Reply:
In the previous lines, if there is no `accountId` it will redirect to `PASSWORD_RESET_ROOT`